app.js

```diff
@@ -0,0 +1,62 @@
+// Mobile Menu
+const open = document.querySelector('.top-menu');
+const mobileMenu = document.querySelector('.mobile-menu');
+const close = document.querySelector('.cancel');
+
+open.addEventListener('click', () => {
+  mobileMenu.classList.add('show');
+});
+
+close.addEventListener('click', () => {
+  mobileMenu.classList.remove('show');
+});
+
+// Create Dynamic section
+
+const featured = document.querySelector('.featured');
+
+const featuredArray = [
+  {
+    profileImg: 'images/ph1.jpg',
+    personName: 'Ray Chen',
+    personTitle: 'Violinist - America',
+    description: 'Ray has played the violin since 3years.The former winner of the Queen Elizabeth Violin Competition will be perfoming the Mozart concerto',
+  },
+
+  {
+    profileImg: 'images/ph2.jpg',
+    personName: 'Jeniffer Cho',
```

Comment:
- [ ] Kindly add 2 more speakers, currently, there are only 4. The total should be 6. 
- [ ] Kindly add a check background image to all the images in the speaker's section as suggested in the video attached.
- [ ] Kindly display only 2 speakers for the mobile version and on clicking more display the remaining.

Reply:
done